src/test/java/us/kbase/test/auth2/lib/AuthenticationLinkTest.java

```diff
@@ -2384,7 +2385,7 @@ public void linkAllFailLinkFailBadTokenOp() throws Exception {
 		final UUID id = UUID.randomUUID();
 		when(storage.getTemporarySessionData(tempToken.getHashedToken())).thenReturn(
 				TemporarySessionData.create(id, NOW, NOW)
-				.login(set(REMOTE)))
+				.login(set(REMOTE), MFAStatus.UNKNOWN))
```

Comment:
~unsure of the implications here because I'm not sure about post-link behavior does a user get a new token immediately after they link a new provider? I would guess not, so I'm guessing there is nothing to worry about. Even so, the worst result is that they have unknown MFA status for that session.~

Reply:
These tests are all testing that someone trying to link an identity using temporary session data from a login flow fails. There's not tokens actually created